clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -473,6 +471,10 @@ public void recordAssembledValueSize(long bytes, long currentTimeMs) {
     assembledValueSizeSensor.record(bytes, currentTimeMs);
   }
 
+  public void recordAssembledRecordSizeRatio(double ratio, long currentTimeMs) {
+    assembledRecordSizeRatioSensor.record(ratio, currentTimeMs);
```

Comment:
I think they're actually the same:
```java
  /**
     * Record a value at a known time. This method is slightly faster than {@link #record(double)} since it will reuse
     * the time stamp.
     */
    public void record(double value, long timeMs) {
```

Reply:
Yes, but it adds the overhead of keeping track of a time in layers above and have to carry it around?

Reply:
I feel like it's widely used under this `StoreIngestionTask#internalProcessConsumerRecord()` method, and since the variable is there, I may as well use it.

I don't have a strong preference for either way.